doc/manual/migrating_from_eeglab.rst

```diff
@@ -0,0 +1,68 @@
+Caveats coming from EEGLAB
+==========================
+
+EEGLAB vs. MNE: Pros and Cons
+-----------------------------
+
+Major differences: MNE-Python ...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* is **Python based**. Python is a fully featured, general programming language. It is easy to learn, easy to read (and thus share, criticize and reproduce code with). It is also free - unlike MATLAB, which you need for EEGLAB, and the MATLAB toolboxes. For example, with MNE, you will never run out of/lack parallel processing toolboxes, and thus MNE more readily uses multiple cores (usually with the `n_jobs` keyword).
```

Comment:
I would link "easy to learn" to this page: https://learnxinyminutes.com/docs/python/